<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player_widget_form</class>
 <widget class="QWidget" name="Player_widget_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="folder_button_">
     <property name="toolTip">
      <string>Open log file</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Urg_viewer.qrc">
       <normaloff>:/icons/folder_icon</normaloff>:/icons/folder_icon</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="play_button_">
     <property name="toolTip">
      <string>Play</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Urg_viewer.qrc">
       <normaloff>:/icons/play_icon</normaloff>:/icons/play_icon</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pause_button_">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Pause</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Urg_viewer.qrc">
       <normaloff>:/icons/pause_icon</normaloff>:/icons/pause_icon</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="stop_button_">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Stop</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Urg_viewer.qrc">
       <normaloff>:/icons/stop_icon</normaloff>:/icons/stop_icon</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="progress_slider_">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="current_label_">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="slash_label_">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="total_label_">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Urg_viewer.qrc"/>
 </resources>
 <connections/>
</ui>
